packages/browser/README.md

```diff
@@ -0,0 +1,170 @@
+# Backtrace Browser
+Backtrace error reporting tool for browser side JavaScript
+ 
+## Table of Contents
+1. [Integration Example](#integration-example)
+1. [Features](#features)
+    - [File Attachments](#file-attachments)
+    - [Metrics](#metrics)
+    - [Breadcrumbs](#breadcrumbs)
+    - [Callback Methods](#callback-methods)
+1. [Documentation](#documentation)
```

Comment:
Thinking about an alt layout that divides between SDK implementation and Backtrace feature implementation...
* SDK Documentation
  * Initial integration - Getting your first errors
  * BacktraceClient configuration
  * Error report modification _(callback methods here)_
* Error Reporting features
  * Attributes
  * Breadcrumbs
  * File attachments
  * Application stability metrics